tests/modules/update.py

```diff
@@ -329,6 +335,12 @@ def test_update_only_show_differences_when_patch(self, mock_prompt):
     modules_json = ModulesJson(self.pipeline_dir)
     update_obj = ModuleUpdate(self.pipeline_dir, update_all=True, show_diff=True)
 
+    # Update modules to a fixed old SHA
+    update_old = ModuleUpdate(
+        self.pipeline_dir, update_all=True, show_diff=False, sha="5e34754d42cd2d5d248ca8673c0a53cdf5624905"
+    )
+    update_old.update()
```

Comment:
Is your thinking here, that this change will be tested when we look at the modules.json? Should we maybe 
explicitly test that the correct  linting test fails?

Reply:
This test makes sure that when we want to only show the differences but not update the module, it is not updated. Specially when there is a patch file because there was a bug on that edge case.
Before updating the modules in the template, we had an old version, so these two new lines were not necessary. Now, as we have the latest version, we need to revert to an older SHA, to be able to test the update afterwards.